### ISSUE TITLE
Typo Term 1 Lesson 4 Introduction to Neural Networks

### ISSUE THREAD

**Issue by GemmaLaraSavill** [udacity/sdc-issue-reports#795]
5. Linear to Logistic Regression

Last sentence: 
Linear regression will lead to neural networks
maybe should be Logistic regression

**Comment by GemmaLaraSavill** [comment]
Not so sure anymore, depends on the context maybe.

**Comment by GemmaLaraSavill** [comment]
Probably both are fine. Please close.

**Comment by baumanab** [comment]
Thanks for bringing this up.  It does seem like both would work and it certainly does depend on the context.  Essentially we can solve both types of regression problems with NNs, the main difference is how we handle network output.  For continuous outputs (linear regression) we don't activate the final output.  For discrete outputs (logistic regression) we activate the final output via sigmoid (binary classification) or softmax (multi-classification).
